README.md

```diff
@@ -14,8 +14,17 @@ On release, automated continuous integration tests run the pipeline on a full-si
 
 <!-- TODO nf-core: Fill in short bullet-pointed list of the default steps in the pipeline -->
 
-1. Read QC ([`FastQC`](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/))
-2. Present QC for raw reads ([`MultiQC`](http://multiqc.info/))
+1. Summarize proteome [`BUSCO`](https://busco.ezlab.org/) completeness at both taxon-specific level, and compared to Eukaryota
```

Comment:
No worries, keep the questions coming! This all is great. The workflow as it stands is by no means perfect or optimized - some aspects (which I see below you've caught and commented on) are a bit hacky and the result of my not being able to find alternative solutions. I'll comment on them in more detail as they arise. 

Responses point by point...

1.  Duplicate files: 
- I would also love to avoid this, and I suspect there is a way to avoid this, but I wasn't able to find one yet. The issue comes from Orthofinder forcing the generation of a directory structure on whatever data is analyzed, and where that data is stored. So, if the test and full dataset are the same and stored in the same place, it will lead to filepath conflicts. 
- Perhaps when running this on tower and downloading data directly from S3 it will be possible to pull the full dataset into one directory and the test data into another, and then we can proceed as normal? I don't have great intuition yet about how this all interfaces with S3/tower. 

2.  nf-core modules: 
- I did end up modifying a number of these to as to better interface with other downstream modules (e.g. passing metadata, dealing with orthofinder induced directory structure, etc).
- Given that I modified these, would it be best practice/appropriate to move them out of the nf-core module directory since they're no longer the same as the version that's distributed?

3.  testout:
- There is no purpose to this and it should be deleted! That's a scrap file from some of my earliest efforts that I forgot to clean up. 

4.  No worries about not extensively editing the nextflow code! It's definitely a pain, but I'm glad you found the comments helpful - I hoped the would be, and they're also just as much intended for me so I can remember what everything's doing!